### PR TITLE
Tentative to exclude macOS CI on nightly

### PR DESCRIPTION
**Pull request by Azzaare**

### CODE REVIEW

**Comment by Azzaare** [comment]
OK, it seems to work fine (in the sense that the CI of Julia nightly on macOS is not in the CI list).
We can probably merge as soon as the CI has ended.

**Review by arubhardwaj — APPROVED** [review]
Awesome @Azzaare. You helped me in maintaining my sanity haha. I will merge it once #17 is approved and merged

**Pull request merged**
